### PR TITLE
validate: add further commands

### PR DESCRIPTION
**Pull request by joshmoore**
* [x] `validate codebook`
 * [x] `validate fov`
 * [x] `validate manifest`

Name change requests welcome.

### CODE REVIEW

File: sptx_format/cli.py [hunk 1]
```diff
@@ -32,14 +32,65 @@ def validate(ctx, experiment_json, fuzz):
...
+
+@validate.command()
+@click.argument("file", metavar="JSON_FILE_OR_URL")
+@click.option("--fuzz", is_flag=True)
+@click.pass_context
+def fov(ctx, file, fuzz):
+    """validate field-of-view against the jsonschemas"""
+
+    try:
+        valid = validate_sptx.validate_file(file, "field_of_view/field_of_view.json", fuzz)
```
Comment (ttung) [inline sptx_format/cli.py]: tiny nit: should avoid using python keywords as variable names.

File: sptx_format/test_experiment.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import pytest
 from pkg_resources import resource_filename
 
+import starfish  # noqa
```
Comment (ttung) [inline sptx_format/test_experiment.py]: what's this for?

Comment (joshmoore) [reply sptx_format/test_experiment.py]: There's an import recursion somewhere between `starfish` and `sptx_format` that shows up in the `sptx_format` tests if importing relatively without first importing `starfish`.

File: sptx_format/cli.py (hunk 1 above)
Comment (joshmoore) [reply sptx_format/cli.py]: https://github.com/spacetx/starfish/pull/1062

**Pull request merged**